docs/pages/admin-guides/access-controls/access-lists/nested-access-lists.mdx

```diff
@@ -0,0 +1,87 @@
+- Verify inherited permissions granted through the nested Access List
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- A user with the `editor` role or equivalent permissions to create and manage Access Lists.
+- Familiarity with basic Access List concepts (see the [Getting Started with Access Lists guide](./guide.mdx)).
+- At least one user with only the `requester` to add to the Access List.
+- At least one application or resource to grant access to.
+
+## What Are Nested Access Lists?
+
+Nested Access Lists allow inclusion of an Access List as a member or owner of another Access List.
+This enables hierarchical permission structures where permissions can be inherited from multiple levels of
+parent Access Lists.
+
+### How Inheritance Works
+
+Let's break down inheritance in Access Lists. Imagine two Access Lists you might have in an organization:
+'Engineering Team' and 'Production Access'. 'Engineering Team' represents a group of engineers, while
+'Production Access' is a higher-level Access List that grants access to production resources.
+
+- **Membership Inheritance**: If 'Engineering Team' is added as a member of 'Production Access',
+all users who are members of 'Engineering Team' inherit member grants (roles and traits) from 'Production Access'.
+- **Ownership Inheritance**: If 'Engineering Team' is added as an owner of 'Production Access',
+all users who are members of 'Engineering Team' inherit owner grants (roles and traits) from 'Production Access', and
+can perform owner actions, such as modifying it or managing its members.
+
+Inheritance is recursive – members of 'Engineering Team' can themselves be Access Lists
+with their own members, and so on. However, circular nesting is not allowed, and nesting is limited
+to a maximum depth of 10 levels.
+
+For more information, see the [Access Lists reference](../../../reference/access-controls/access-lists.mdx).
+
+## Creating a Nested Access List
+
+Now, let's walk through creating a nested Access List and establishing inheritance. In this example, we'll
+create a child Access List, 'Engineering Team', which inherits permissions from a parent, 'Production Access'.
+
+### Step 1/3. Create Child Access List
+
+In the Teleport Web UI, go to the "Identity" tab and select "Access Lists" from the sidebar.
+Click on "Create New Access List", and fill in the details:
+
+- **Title**: Engineering Team
+- **Deadline for First Review**: Select a future date.
+- **Member Grants**: Leave this empty, as the list will inherit the parent's member grants.
+- **Owners**: Add yourself or any appropriate users as owners.
+- **Members**: Add users who should be part of this Access List, such as `test-user`.
+
+Click "Create Access List" to save the Access List.
+
+### Step 2/3. Create Parent Access List
+
+From the "Access Lists" page, click on "Create New Access List" and fill in the details for our parent list:
```

Comment:
Some paragraphs use single-quotes, while others use double-quotes. We should be consistent here.